auth/indieauth/response.go

```diff
@@ -0,0 +1,16 @@
+package indieauth
+
+// Profile represents optional profile data that is returned
+// when completing the IndieAuth flow.
+type Profile struct {
+	Name  string `json:"name"`
+	URL   string `json:"url"`
+	Photo string `json:"photo"`
+}
+
+// Response the response returned when completing
+// the IndieAuth flow.
+type Response struct {
+	Me      string  `json:"me"`
+	Profile Profile `json:"profile"`
```

Comment:
this may not always be returned - what happens if it's absent?

Reply:
The profile isn't required. I might as well just remove it.

Reply:
Oh sorry, misread, you're looking at our server response.

There's currently no checks for these kind of things put in. That still needs to happen. IndieAuth support (client or server) will not be available unless you've set the required configuration (URL mostly). Errors need to be surfaced in the case that it's not configured.